docs/user/managing-devices.md

```diff
@@ -0,0 +1,558 @@
+
+| Status Field | Description |
+| ------------ | ----------- |
+| | |
+
+### Managing Applications on the Web UI
+
+### Managing Applications on the CLI
+
+To deploy an application package from an OCI registry, specify it in the device's `spec.applications[]` as follows:
+
+```yaml
+apiVersion: v1alpha1
+kind: Device
+metadata:
+  name: some_device_name
+spec:
+[...]
+  applications:
+  - name: wordpress
+    image: quay.io/flightctl-demos/wordpress-app:latest
+    envVars:
+      WORDPRESS_DB_HOST: "mysql"
+      WORDPRESS_DB_USER: "user"
+      WORDPRESS_DB_PASSWORD: "password"
+[...]
+```
+
+To deploy an unpackaged application from a Git repository, specify it in the device's `spec.applications[]` as follows:
+
+```yaml
+apiVersion: v1alpha1
+kind: Device
+metadata:
+  name: some_device_name
+spec:
+[...]
+  applications:
+  - name: wordpress
+    git:
+      url: https://github.com/flightctl/flightctl-demos.git
+      revision: v1.0
+      path: /wordpress
+    envVars:
+      WORDPRESS_DB_HOST: "mysql"
+      WORDPRESS_DB_USER: "user"
+      WORDPRESS_DB_PASSWORD: "password"
+[...]
+```
+
+To deploy an unpackaged application inline with the device specification, specify it in the device's `spec.applications[]` as follows:
+
+```yaml
+apiVersion: v1alpha1
+kind: Device
+metadata:
+  name: some_device_name
+spec:
+[...]
+  applications:
+  - name: wordpress
+    inline:
+      podman-compose.yaml: |
+        version: “3.7"
+        services:
+          wordpress:
+            image: “wordpress:latest”
+        [...]
+    envVars:
+      WORDPRESS_DB_HOST: "mysql"
+      WORDPRESS_DB_USER: "user"
+      WORDPRESS_DB_PASSWORD: "password"
+[...]
+```
+
+## Using Device Lifecycle Hooks
+
+## Monitoring Device Resources
+
+You can set up monitors for device resources and define alerts when the utilization of these resources crosses a defined threshold. When the agent alerts the Flight Control service, the service sets the device status to "degraded" or "error" (depending on the severity level) and may suspend the rollout of updates and alarm the user as a result.
+
+Note this is not meant to replace an observability solution. If your use case requires streaming logs and metrics from devices into an observability stack and the device's network bandwidth allows this, see [Adding Device Observability](adding-device-observability.md) for ways to approach that.
+
+Resource monitors take the following parameters:
+
+| Parameter | Description |
+| --------- | ----------- |
+| MonitorType | The resource to monitor. Currently supported resources are "CPU", "Memory", and "Disk". **[TODO: Check whether the "Custom" resource type is implemented.]** |
+| SamplingInterval | The interval in which the monitor samples utilization, specified as positive integer followed by a time unit ('s' for seconds, 'm' for minutes, 'h' for hours, 'd' for days). |
```

Comment:
There is a bug here in the agent as we can not convert d to days. Tracked via https://issues.redhat.com/browse/EDM-518

Reply:
I commented on EDM-518 but I can submit a PR for that based on the logic in #529 .